lms/templates/index.html

```diff
@@ -54,12 +54,17 @@
       % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
         <section class="courses">
             <ul class="courses-listing">
-            %for course in courses:
+            %for course in courses[:9]:
               <li class="courses-listing-item">
                 <%include file="course.html" args="course=course" />
               </li>
             %endfor
             </ul>
+            <div class="more-courses">
+              <a class="more-link-arrow" title="View all Courses" href="${marketing_link('COURSES')}" >
```

Comment:
The ``title`` attribute is not needed here since it has the same value as the actual element's text. The text is preferable as it is already i8tn friendly.